app/assets/stylesheets/sign_in.scss

```diff
@@ -0,0 +1,9 @@
+body#sign-in {
+  background-color: #4b5c5c;
```

Comment:
Color literals like `#4b5c5c` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background, background-color